### PR TITLE
feat: Update app name on topbar based on version

### PR DESCRIPTION
**Pull request by mohandast52**
Screenshot

### CODE REVIEW

File: frontend/components/Layout/TopBar.tsx [hunk 1]
```diff
@@ -56,7 +59,10 @@ export const TopBar = () => {
         <DisabledLight />
       </TrafficLights>
 
-      <Text>Pearl (alpha)</Text>
+      <Text>
+        Pearl (alpha)
+        {store?.storeState?.appVersion?.includes('rc') ? '' : ' (staging)'}
```
Comment (Tanya-atatakai) [inline frontend/components/Layout/TopBar.tsx:64]: won't we be able to get it from `process.env.IS_STAGING`?

File: electron/constants/publishOptions.js [hunk 2]
```diff
@@ -5,8 +5,7 @@ const publishOptions = {
   token: process.env.GH_TOKEN,
   private: false,
   publishAutoUpdate: true,
-  releaseType: process.env.IS_STAGING === 'true' ? 'prerelease' : 'release',
-  channel: process.env.IS_STAGING === 'true' ? 'alpha' : 'latest',
+  releaseType: process.env.IS_STAGING === 'true' ? 'draft' : 'release',
```
Comment (truemiller) [inline electron/constants/publishOptions.js:8]: no longer need this, we will always release to `draft` then promote to full release
```suggestion
  releaseType: 'draft',
```

File: electron/install.js [hunk 3]
```diff
@@ -7,6 +7,11 @@ const process = require('process');
 const { spawnSync } = require('child_process');
 const Docker = require('dockerode');
 
+/**
+ * current version of the pearl release
+ * - use "" (nothing as a suffix) for latest release candidate, for example "0.1.0rc26"
+ * - use "alpha" for alpha release, for example "0.1.0rc26-alpha"
+ */
 const Version = '0.1.0rc26';
```
Comment (truemiller) [inline electron/install.js:15]: not your change but could you refactor this to: `const OlasMiddlewareVersion`
should avoid confusion for others in future

File: frontend/components/Layout/TopBar.tsx [hunk 4]
```diff
@@ -48,6 +50,19 @@ const TopBarContainer = styled.div`
 
 export const TopBar = () => {
   const electronApi = useElectronApi();
+  const store = useStore();
+
+  const versionName = useMemo(() => {
+    const releaseType = store?.storeState?.releaseType;
+    if (releaseType === 'draft') return ' (staging)';
```
[the remark this answers is not in the document]
Comment (truemiller) [reply frontend/components/Layout/TopBar.tsx:57]: yeah, not sure we need appVersion is store..
can probably just a expose constant through electron api

**Review by angrybayblade — APPROVED** [review]

**Review by truemiller — APPROVED** [review]

**Pull request merged**